src/components/ChatEntry.jsx

```diff
@@ -1,20 +1,30 @@
 import './ChatEntry.css';
+import TimeStamp from './TimeStamp';
+import PropTypes from 'prop-types';
 
-const ChatEntry = () => {
+const ChatEntry = (props) => {
   return (
-    <div className="chat-entry local">
-      <h2 className="entry-name">Replace with name of sender</h2>
+    <div className={`chat-entry ${props.side}`}>
+      <h2 className="entry-name">{props.sender}</h2>
       <section className="entry-bubble">
-        <p>Replace with body of ChatEntry</p>
-        <p className="entry-time">Replace with TimeStamp component</p>
-        <button className="like">🤍</button>
+        <p>{props.body}</p>
+        <p className="entry-time">{props.timeStamp}</p>
```

Comment:
@Shirley425, you are failing a single test because you didn't utilize the `Timestamp` component.  You could have fixed this with a single line like so: 
```suggestion
        <TimeStamp time={props.timeStamp}/>
```
In industry, you will want to make sure that you are passing all tests before creating a PR. Especially for something like this. It might give off the impression that you aren't paying close attention to the code that you are working on.